SwiftyBluetooth/Source/Central.swift

```diff
@@ -144,6 +144,21 @@ extension Central {
         return self.centralProxy.centralManager.isScanning
     }
     
+    /// The underlying CBCentralManager delegate value
+    public var delegate: CBCentralManagerDelegate? {
```

Comment:
@tehjord definitely makes sense why you would want to keep this the way it is--it definitely is a trade-off between control and safety. The reason I wanted to make this change was that for our purposes when performing an over-the-air firmware update on our peripheral, we need to pass our central manager delegate to a different framework that handles a lot of the firmware update details and then return the delegate control to the application once the update has finished.

For our purposes we actually would need this change as well: https://github.com/jakerockland/SwiftyBluetooth/tree/centralManager-access 😅 

Can understand not wanting to merge this change though--happy to accomplish this on our own fork or perhaps just using an extension.